### DISCUSSION TITLE
PCAP pre-populated dropdowns

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#1967] [Ideas] [answered]
Would it be possible to add pre-populated values (~last five, last fifteen minutes) to the begin and end date filters in the PCAP add job interface? Perhaps a dynamically generated drop down on the Sensor ID field as well?

**Comment by dougburks** [comment, marked as answer]
Thanks for the idea. We will take this into consideration.
